### PR TITLE
Overhaul interface component

### PR DESCRIPTION
**Pull request by mharis001**
- Overhaul `interface` component with new display
    - Add `description` and `picture` config entrys to `CfgRadioStations`
- Some general code cleanup

<details>
<summary>Image</summary>

radio_ui_2

</details>

### CODE REVIEW

File: addons/interface/XEH_postInit.sqf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 
-if !(isClass(configFile >> "CfgPatches" >> "ace_interaction")) then {
+if (!isClass (configFile >> "CfgPatches" >> "ace_interaction")) then {
```
Comment (BrettMayson) [inline addons/interface/XEH_postInit.sqf:3]: I know this is standard in ACE and other projects, still not sure how I feel about it though

File: addons/interface/functions/script_component.hpp [hunk 2]
```diff
@@ -1 +1 @@
-#include "../script_component.hpp"
+#include "\dynulo\radio\addons\interface\script_component.hpp"
```
Comment (BrettMayson) [inline addons/interface/functions/script_component.hpp:1]: Is there a reason behind this change?

File: addons/interface/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="Radio">
+    <Package name="Interface">
+        <Key ID="STR_Radio_Interface_DisplayName">
+            <English>FM Radio</English>
+        </Key>
+        <Key ID="STR_Radio_Interface_On">
+            <English>On</English>
```
Comment (BrettMayson) [inline addons/interface/stringtable.xml:8]: Can use `str_a3_rscdisplayconfigure_ca_mouseacceleration_on` instead

File: addons/interface/stringtable.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        </Key>
+        <Key ID="STR_Radio_Interface_Off">
+            <English>Off</English>
```
Comment (BrettMayson) [inline addons/interface/stringtable.xml:11]: Can use `str_a3_rscdisplayconfigure_ca_mouseacceleration_off` instead

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: addons/interface/functions/script_component.hpp (hunk 2 above)
Comment (mharis001) [reply addons/interface/functions/script_component.hpp:1]: Convention mostly from ACE-like projects. The `manager` component also has it like this.

**Pull request merged**